data/SJ-Kumar.json

```diff
@@ -0,0 +1,129 @@
+{
+   "basics": {
+    "name": "S.Jayanth Kumar",
+    "label": "BTech UnderGraduate at VIT Chennai",
+    "image": "https://github.com/SJ-Kumar/SJ-Kumar/blob/main/Jayanth.jpg",
```

Comment:
```diff
- "image": "https://github.com/SJ-Kumar/SJ-Kumar/blob/main/Jayanth.jpg",
+ 
```

Remote the line 5 image or Avatar will auto update with your github profile

Reply:
I had replaced with the updated version. Please verify the latest version.